tests/meson.build

```diff
@@ -0,0 +1,82 @@
+test_i18n_exe = executable(
+  'test-i18n',
+  'test-i18n.c',
+  dependencies: libgphoto2_dep,
+)
```

Comment:
If you add `build_by_default: false` then you can skip building these test programs when just running `meson setup && ninja && ninja install` and not ever running `ninja test`. They will automatically be built as part of `ninja test` though.